<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferList</class>
 <widget class="QWidget" name="TransferList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="upload">
      <attribute name="title">
       <string>上传列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="upload_scroll">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>394</width>
            <height>256</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="download">
      <attribute name="title">
       <string>下载列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="download_scroll">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>394</width>
            <height>256</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="record">
      <attribute name="title">
       <string>传输记录</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="record_scroll">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>394</width>
            <height>256</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>293</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="clear_btn">
                <property name="text">
                 <string>清除记录</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="record_msg"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
